flow-sample-orientation-lock/src/main/java/flow/sample/orientation/LandscapeScreen.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2016 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package flow.sample.orientation;
+
+final class LandscapeScreen {
```

Comment:
maybe make a Screen interface w/ `getLayout()` and `@Nullable int getRequestedOrientation()` (w/ values of `Configuration.ORIENTATION_*`), to tidy up the dispatcher a bit.